.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Default version to use when preparing a release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Default version to use for new local working copy."
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.BUILD_SVC_PAT }}
```

Comment:
What's this new token?

Reply:
This is for being able to push changes in pom files without approval, actually not needed for this PR.

Reply:
I remove it for now :)